src/main/java/com/google/sps/data/Listing.java

```diff
@@ -0,0 +1,48 @@
+package com.google.sps.data;
+
+/** Class for Listing object with the specified attributes returned from the places API **/
+public class Listing{
+    private String name;
+    private MapLocation mapLocation;
+    private int rating;
+    private String image;
+    private String url;
+
+  /**
+  * Creates a new Listing
+  * @param name Name of business
+  * @param mapLocation Location of business (lat/long coordinate)
+  * @param rating Numerical rating of business (1-5)
+  * @param image Image
+  * @param url Url to business' site
+  * @return Listing object
+  **/
+
+  public Listing(String name, MapLocation mapLocation, int rating, String image, String url) {
+    this.name = name;
+    this.mapLocation = mapLocation;
+    this.rating = rating;
+    this.image = image;
+    this.url = url;
+  }
```

Comment:
I might add a section for a business description. Not sure if we can pull that from the Places API in the long run, do you?

Reply:
The Places API does not return a business description, but it does return a "type" like food and establishment which signifies that the entity is a business which might be even more helpful down the road. I added a businessType parameter to account for that. I didn't see anything regarding business description on the places API, but that's definitely something we can add in the future if we find a way to get that info.

Reply:
Works for me, thx for the explanation

Reply:
+1